index.html

```diff
@@ -18,7 +18,7 @@ <h1 class="tagline col-md-5">
             <i class="fa fa-plus"></i>
             <img src="{{ site.baseurl }}/assets/images/foreman.png" alt="Foreman"/>
           </p>
-          <a href="{{ site.baseurl }}/docs/2.0/installation/index.html" class="get-started">
+          <a href="{{ site.baseurl }}/docs/{{ site.latest }}/installation/index.html" class="get-started">
```

Comment:
I would just hardcode this to '2.1'.  Mainly so we can add 2.2 docs without users clicking 'get started' and being sent to 2.2